doc/source/ecosystem.rst

```diff
@@ -129,6 +129,22 @@ both "column wise min/max and global min/max coloring."
 API
 -----
 
+`pandas-datareader <https://github.com/pydata/pandas-datareader>`_
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+pandas-datareader is a remote data access library for pandas. ``pandas.io`` from pandas < 0.18 is now refactored to and importable from ``pandas_datareader`` (PyPI:``pandas-datareader``). Many/most of the supported APIs have at least a documentation paragraph in the `pandas-datareader docs <https://pandas-datareader.readthedocs.org/en/latest/>`_:
```

Comment:
- [ ] so `>= 0.17.0`
On Apr 2, 2016 10:26 AM, "Jeff Reback" <notifications@github.com> wrote:

> In doc/source/ecosystem.rst
> <https://github.com/pydata/pandas/pull/12773#discussion_r58294798>:
>
> > @@ -129,6 +129,22 @@ both "column wise min/max and global min/max coloring."
> >  API
> >  -----
> >
> > +`pandas-datareader <https://github.com/pydata/pandas-datareader>`_
> > +~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
> > +pandas-datareader is a remote data access library for pandas. ``pandas.io`` from pandas < 0.18 is now refactored to and importable from ``pandas_datareader`` (PyPI:``pandas-datareader``). Many/most of the supported APIs have at least a documentation paragraph in the `pandas-datareader docs <https://pandas-datareader.readthedocs.org/en/latest/>`_:
>
> this was split off as of 0.17.0.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/pydata/pandas/pull/12773/files/3b11bfab00bc61aec9330ae75bbec77ec56fff15#r58294798>
>